### PR TITLE
 Use consistent type of IDs for all the modules within framework - Closes #7090 

### LINKED ISSUE #7090: Use consistent type of IDs for all the modules within framework
### Description

We are using bytes or number type for ID (chainID, moduleID, commandID, etc) inconsistently across the all the modules. we should use one type for IDs consistently.

Conclusion, we use IDs as bytes consistently in the Framework which means that all the schemas, interfaces within the framework needs to be updated.

### Acceptance Criteria

All the IDs are using one type consistently across all the modules within framework

### PR DESCRIPTION
**Pull request by Incede**
### What was the problem?

This PR resolves #7090 

### How was it solved?

* Update formats as specified
* Fix min fee bug
* Update min fees in fixtures to account for the transaction size with new formats
*  Update commander to process new input format (WIP)

### How was it tested?

Updated tests (Commander tests TBD)

### CODE REVIEW

File: framework/src/modules/fee/module.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { FeeEndpoint } from './endpoint';
 import { configSchema } from './schemas';
 
 export class FeeModule extends BaseModule {
-	public id = MODULE_ID_FEE;
+	public id = intToBuffer(MODULE_ID_FEE, 4);
```
[the remark this answers is not in the document]
Comment (Incede) [reply framework/src/modules/fee/module.ts:32]: I think we were using `intToBuffer` in many places in the code before so probably it's okay to use directly.

File: framework/src/modules/dpos_v2/module.ts [hunk 2]
```diff
@@ -663,7 +665,7 @@ export class DPoSModule extends BaseModule {
 			generators,
 		);
 
-		const delegateStore = getStore(MODULE_ID_DPOS, STORE_PREFIX_DELEGATE);
+		const delegateStore = getStore(MODULE_ID_DPOS_BUFFER, STORE_PREFIX_DELEGATE);
```
[the remark this answers is not in the document]
Comment (Incede) [reply framework/src/modules/dpos_v2/module.ts:668]: how is it related to the current issue ?

File: framework/src/modules/fee/module.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ishantiw) [reply framework/src/modules/fee/module.ts:32]: To be tackled in https://github.com/LiskHQ/lisk-sdk/issues/7280

File: framework/src/modules/interoperability/base_interoperability_store.ts [hunk 3]
```diff
@@ -157,31 +150,40 @@ export abstract class BaseInteroperabilityStore {
...
-		const channelSubstore = this.getStore(MODULE_ID_INTEROPERABILITY, STORE_PREFIX_CHANNEL_DATA);
+		const channelSubstore = this.getStore(
+			getIDAsKeyForStore(MODULE_ID_INTEROPERABILITY),
+			STORE_PREFIX_CHANNEL_DATA,
+		);
 		const channel = await channelSubstore.getWithSchema<ChannelData>(chainID, channelSchema);
 
 		const outboxRootSubstore = this.getStore(
-			MODULE_ID_INTEROPERABILITY,
+			getIDAsKeyForStore(MODULE_ID_INTEROPERABILITY),
```
[the remark this answers is not in the document]
Comment (ishantiw) [reply framework/src/modules/interoperability/base_interoperability_store.ts:160]: To be tackled in https://github.com/LiskHQ/lisk-sdk/issues/7280

File: framework/src/modules/dpos_v2/module.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ishantiw) [reply framework/src/modules/dpos_v2/module.ts:668]: To be tackled in https://github.com/LiskHQ/lisk-sdk/issues/7280

**Review by ishantiw — APPROVED** [review]
Nice work @Incede 🏅

**Review by mitsuaki-u — APPROVED** [review]
🚀 🚀

**Pull request merged**